vault/dendron.topic.schema.cook.first-schema.md

```diff
@@ -0,0 +1,171 @@
+---
+id: 5U4eAiqshI67VxIL40KWH
+title: Making Your First Schema
+desc: 'Learn to harness the powers of schema and automatically insert templates into new notes'
+updated: 1640789533041
+created: 1640784965050
+---
+
+## Summary
+
+[[Schemas|dendron://dendron.dendron-site/dendron.topic.schema]] make management of new notes easier by automatically inserting Markdown [[templates|dendron://dendron.dendron-site/dendron.topic.templates]] into new notes. This guide will walkthrough making a template for daily journaling, using [The Five Minute Journal](https://www.intelligentchange.com/products/the-five-minute-journal) method, and using schema to handle where the template will be used in your [[hierarchy|dendron://dendron.dendron-site/dendron.topic.hierarchies]].
+
+We'd like all daily journals to have a child note for our 5MJ:
+
+- `daily.journal.yyyy.mm.dd`: Logging tasks, and anything else we'd like in our daily journal. Dendron can automatically make these with the [[Daily Journal|dendron.topic.special-notes#daily-journal]] command (`Ctrl+Shift+I` / `Cmd+Shift+I`).
```

Comment:
for these tutorials, it would make sense to have a concepts section that introduces what we're talking about
- eg. Daily Journal, Create Daily Journal Command, Schemas, Templates